### PR TITLE
Update GitHub Copilot.json

### PR DESCRIPTION
**Pull request by bricebai1**
Changing a brittle selector that broke down with the site update. (The new selector that I've found is not that simple either. I would be happy to learn if there is a simpler one.)

### CODE REVIEW

File: declarations/GitHub Copilot.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
           "fetch": "https://docs.github.com/en/site-policy/github-terms/github-terms-of-service",
           "select": [
             "#title-h1",
-            ".MarkdownContent_markdownBody__7_8Gt"
+            ".ArticleGridLayout_containerBox__lLLio"
```
Comment (MattiSG) [inline declarations/GitHub Copilot.json:10]:
```suggestion
            ".markdown-body"
```

**Review by MattiSG — COMMENTED** [review]
Thanks @bricebai1! There is indeed a simpler selector, as suggested 🙂 

When inspecting the document, you can see that the node with the legal text has the following `class` attribute: `class="MarkdownContent_markdownBody__v5MYy markdown-body"`.
As the classes are separated by a space, this means that the node has both the `MarkdownContent_markdownBody__v5MYy` and `markdown-body` classes. It is thus more stable to use the second one 🙂 

I will apply this change and proceed with merging.

**Review by MattiSG — APPROVED** [review]

**Review by MattiSG — CHANGES_REQUESTED** [review]
Oops, sorry, I was too fast in approving. We need to make sure that the `.markdown-body` was already valid before. If not (or if we don't check that), we should add a [history entry](https://docs.opentermsarchive.org/contributing-terms/#service-history).

**Comment by MattiSG** [comment]
The history entry can be generated automatically by using the contribution tool.

**Comment by clementbiron** [comment]
GitHub Copilot Terms of Service is using multipage which is not supported by the contribution tool.

**Review by clementbiron — APPROVED** [review]
It seems that the `.markdown-body` selector has been valid since the [first snapshot](https://github.com/OpenTermsArchive/GenAI-snapshots/commit/7939ce45cda7374137a57f5b48690065296f73d0) was recorded.

**Pull request merged**